### ISSUE TITLE
How to uninstall it?

### ISSUE THREAD

**Issue by brunojustino** [helmetjs/helmet#265]
I installed using: "npm install helmet". After that I uninstalled using: "npm uninstall helmet"

After that my site doesent work anymore. It keeps giving me the same errors as when helmet was installed. Like: refused to load script, refused to load image, uncaught typerror, etc.

**Comment by EvanHahn** [comment]
It seems like you uninstalled it correctly. What errors are you seeing? Are you still importing `helmet` anywhere? Is an old version of your app running?

By the way, some of those errors may have been a result of Helmet's default Content Security Policy, which you can disable like this:

```js
app.use(helmet({
  contentSecurityPolicy: false,
}));
```

**Comment by brunojustino** [comment]
The errors is the ones I've said. It shows on the console. I also tried reinstalling helmet and setting contentsecuritypolicy: false, but it didnt wortk either. The solution I found it was to set myself in the header the content policies. I put a very permissive one and got my website working again.
I think is that after the first app.use(helmet()) there no turning back after this. Even if you uninstall it the policies where already applyied so uninstalling wont make any difference. Maybe this is something to consider when installing helmet, put a little warn that if you not an expert there's no way to revert some changes, when uninstalling it

**Comment by EvanHahn** [comment]
I'm surprised that you're seeing that. Did you try a hard refresh in your browser? Is it possible that something is being cached somewhere? Is this happening in development or production (or both)?

**Comment by brunojustino** [comment]
This is happening in development on my localhost where I have a lot of scripts from others sources: bootstrap, jquery, googlefonts, etc. and some images direct from internet. Theses things all stopped working even after I uninstalled helmet. I tried different browsers with same results. I dont know if uninstalling helmet means redo the policies helmet set but In my case it certainly didnt do anything. So I have to set myself on the html header to make my website work properly again.
